R/rd-example.R

```diff
@@ -106,11 +106,19 @@ highlight_examples <- function(code, topic, env = globalenv()) {
     do.call(fig_save, c(list(plot, name), context_get("figures")))
   }
 
-  downlit::evaluate_and_highlight(
-    code,
-    fig_save = fig_save_topic,
-    env = child_env(env)
-  )
+  if (requireNamespace("evaluate")) {
+    downlit::evaluate_and_highlight(
+      code,
+      fig_save = fig_save_topic,
+      env = child_env(env),
+      output_handler = evaluate::new_output_handler(value = pkgdown_print)
```

Comment:
Again, I think it's fine to just error here if evaluate isn't installed.

Reply:
And done.